typed-racket-lib/typed-racket/typecheck/internal-forms.rkt

```diff
@@ -111,17 +110,18 @@
 
   (syntax-parse stx
     [(_ :clause ...)
-     (template
-       (begin
-         (begin
-           (define-syntax-class name
-             #:auto-nested-attributes
-             #:literal-sets ((internal-literals #:at name))
-             (pattern i:internal^ #:with (lit . body) #'i.value))
-           (define pred
-             (syntax-parser
-               [(~var _ name) #t]
-               [_ #f]))) ...))]))
+     (syntax
+      (begin 
```

Comment:
Can we retain the spacing here to reduce the diff? I think aside from `template` -> `syntax` this is just indentation.